ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -43,6 +49,25 @@ class NavigationControllerDemoController: DemoController {
         container.addArrangedSubview(createButton(title: "Change the style every second", action: #selector(showSearchChangingStyleEverySecond)))
     }
 
+    let gradient: CAGradientLayer = {
+        let redColor = UIColor(colorValue: ColorValue(0xAE7EE1)).withAlphaComponent(0.4).cgColor
```

Comment:
Since you want a `CGColor` with an alpha value anyway, might be worth just converting the hex value to the rgb floats and using `CGColor(red: green: blue: alpha:)`